src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts

```diff
@@ -73,6 +75,20 @@ class ongoingTasks extends viewModelBase {
     private initObservables() {
         this.myNodeTag(this.clusterManager.localNodeTag());
         this.serverWideBackupUrl = appUrl.forServerWideBackupList();
+        
+        this.taskNameToCount = ko.pureComputed<dictionary<number>>(() => {
+            var tasksNameToCount: { [id: string] : number; } = {};
```

Comment:
never use var in typescript. Const or let.

Reply:
right - can't believe I overlooked that  :)